docs/pages/getting-started.html

```diff
@@ -52,6 +52,10 @@ <h3 class="usa-heading">For developers</h3>
 
 <p>The standards are designed to support both older and newer browsers through progressive enhancement, and officially support Internet Explorer 9 and up, and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage today, and most agency websites should be able to safely begin support at Internet Explorer 9.</p>
```

Comment:
Replace "Internet Explorer 8 and below generally see very low usage today," with "Internet Explorer 8 and below see very low usage,"